packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -209,6 +209,31 @@ class FileCollection extends DocumentCollection {
     return this.statById(id)
   }
 
+  /**
+   * Get all files by their ids
+   *
+   * @param {Array<string>} ids - files ids
+   * @returns {Promise<{data, execution_stats}>} JSON API response
+   */
+  async getAll(ids) {
+    let resp
+    try {
+      resp = await this.stackClient.fetchJSON(
+        'POST',
+        uri`/files/_all_docs?include_docs=true`,
+        {
+          keys: ids
+        }
+      )
+    } catch (error) {
+      return dontThrowNotFoundError(error)
+    }
+    return {
+      data: resp.data.map(f => normalizeFile(f)),
+      execution_stats: resp.execution_stats
```

Comment:
I missed the missing meta, would you mind adding this?
```
meta: {
  count: resp.data.length
}
```
To be consistent with the `getAll` in DocumentCollection
And udpate the return type accordingly

Reply:
corrigé